go/service/chat_local.go

```diff
@@ -251,6 +285,60 @@ func (h *chatLocalHandler) ResolveConversationLocal(ctx context.Context, arg cha
 	}, nil
 }
 
+func (h *chatLocalHandler) resolveConversationsPipeline(ctx context.Context, convs []chat1.Conversation) ([]chat1.ConversationLocal, error) {
+	// Fetch conversation local information in parallel
+	ctx, _ = getUserInfoMapper(ctx, h.G())
+	type jobRes struct {
+		conv  chat1.ConversationLocal
+		index int
+	}
+	type job struct {
+		conv  chat1.Conversation
+		index int
+	}
+	eg, ctx := errgroup.WithContext(ctx)
+	convCh := make(chan job)
+	retCh := make(chan jobRes)
+	eg.Go(func() error {
+		defer close(convCh)
+		for i, conv := range convs {
+			convCh <- job{conv: conv, index: i}
```

Comment:
Actually wait, I don't think that is needed when writing into the channel is it?